app/views/member/details/edit.html.haml

```diff
@@ -11,6 +11,7 @@
         = f.input :surname, required: true
         = f.input :pronouns
         = f.input :email, required: true
+        = f.input :dietary_restrictions, label: 'If you have dietary restrictions, please add them here'
```

Comment:
The label should probably use the translation mechanism rather than a raw English string, to allow it to be translated for non-English chapters. eg.
```suggestion
        = f.input :dietary_restrictions, label: t('member.details.edit.dietary_restrictions')
```

With a corresponding entry in `config/locales/en.yml`.

Someone more familiar with Ruby on Rails might be able to give you more specific guidance.

Reply:
Good shout, done that!